data/coalition/coalition missions.txt

```diff
@@ -1393,3 +1393,281 @@ mission "Arachi Cartoonist 2"
+				accept
+	on complete
+		"coalition jobs" += 3
+		payment 300000
+		conversation
+			`While Ulmodi begins unloading the athletes' luggage and Tumub counts out your payment, Pairri approaches you. She stretches her legs trying to raise herself up your height, and you humor her by crouching down. In response, she offers you a small coin that you don't recognize, then says, "Incapable of arriving on time, a mutual friend of ours was. Available, their ticket is."  The coin has a symbol similar to the old Olympic Games on Earth, except with three interlocked rings instead of five.`
+			`	"Very long and tiring, even to watch, for a first timer the games can be," Tumub explains. "Understand, we would, if too busy, you are."`
+			`	"Still, much joy, it would bring us, if to watch the games, you chose," says Ulmodi. "Rare, such a chance is, to show an outsider the wonders of our games. Good fortune, your presence and support would bring us."`
+			choice
+				`	"Thank you. I would be glad to view the games."`
+				`	"I'm honored, but sadly I have other matters to attend to right now."`
+					decline
+			`	You take the "ticket," and they start explaining how the games work.`
+			`	"Separate competitions, each species has," Tumub says. "To value each one's history, it is so."`
+			`	"A grand, cooperative match, at the end of it all takes place," Ulmodi further details. "Teams consisting of all three species, formed they are. A joyous, unmatched experience, the last spectacle is."`
+			`	Since they need to go their separate ways and prepare for the games, they don't have time to explain each of the individual sports, but assure you that watching them will be enjoyable even if you don't understand the rules. Nearby, you find a Heliarch agent who quickly points you to the line for ticket admission, which you're shocked to see is an individual queue of thousands of people from all three Coalition species. Despite the length, the Heliarchs managing admissions are incredibly efficient and organized; you spend barely ten minutes waiting to get in.`
+			`	After turning in your ticket at the main gate, another Heliarch shows you where you should go for each competition. Three massive transports, similar in design to old-fashioned bullet trains, wait to take spectators to a different arena for each set of games.`
+			`	They also explain that, from there, you'll take another train to go to the "grand spectacle" that serves as a central finale for the Coalition Games once the individual matches are over. You thank them for the explanation and decide on which train to take.`
+			choice
+				`	(Take the train to the Saryd game.)`
+					goto saryd
+				`	(Take the train to the Kimek game.)`
+					goto kimek
+				`	(Take the train to the Arachi game.)`
```

Comment:
```suggestion
				`	(Take the train to the Arach game.)`
```

No 'i' since it's the adjective. You also use the plural form in the conditions while 'saryd' is kept singular, but that's less important.